docs/gatsby-node.js

```diff
@@ -8,7 +8,7 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
 
   if (node.internal.type === `MarkdownRemark`) {
     const slug = createFilePath({ node, getNode, basePath: `pages` });
-    const idName = _.kebabCase(node.frontmatter.title);
+    const idName = `${_.kebabCase(node.frontmatter.title)}`;
```

Comment:
why is that change necessary?

Reply:
oops sorry about that, my first attempt was to prefix octokit- there but I switched to removing octokit- from the elasticsearch config. I'll remove that.